packages/react-scripts/template/README.md

```diff
@@ -22,7 +22,7 @@ You can find the most recent version of this guide [here](https://github.com/fac
 - [Adding a Stylesheet](#adding-a-stylesheet)
 - [Post-Processing CSS](#post-processing-css)
 - [Adding a CSS Preprocessor (Sass, Less etc.)](#adding-a-css-preprocessor-sass-less-etc)
-- [Adding Images and Fonts](#adding-images-and-fonts)
+- [Adding Images, Fonts and Files](#adding-images-fonts-and-files)
```

Comment:
Can we make this `Adding Images, Fonts, and Files` to be consistent with the rest of the docs? It seems that majority of the places use Oxford.